tests/test_render_window_interactor.py

```diff
@@ -10,16 +10,12 @@
     not system_supports_plotting(), reason="Requires system to support plotting"
 )
 
-skip_needs_vtk_9 = pytest.mark.skipif(
-    pyvista.vtk_version_info < (9, 1, 0), reason="Requires VTK>=9.1.0"
-)
-
```

Comment:
This was a weird case where the name is "needs vtk 9" but the check is "needs vtk 9.1". I decided the drop the name and keep the behaviour.

Reply:
Agreed. Thanks!